Extension/src/LanguageServer/client.ts

```diff
@@ -327,10 +327,13 @@ interface PublishRefactorDiagnosticsParams {
 export interface CreateDeclarationOrDefinitionParams {
     uri: string;
     range: Range;
+    copyToClipboard: boolean;
 }
 
 export interface CreateDeclarationOrDefinitionResult {
-    changes: { [key: string]: any[] };
+    edit: any;
```

Comment:
Can the "any" type be avoided?

Reply:
It looks like it should be WorkspaceEdit, right?

Reply:
Technically not because it is nested in a json object, I think I made the json struct incorrectly on the native side but haven't fixed it yet

Reply:
Okay, it seems like I might not be understanding something. Maybe I'll figure it out later.

Reply:
Yeah, it still seems like this should be WorkspaceEdit, such as like the other examples in CodeAnalysisDiagnosticRelatedInformation. I'm not seeing what the difference is or why WorkspaceEdit wouldn't work.

Reply:
Unresolving this comment, as I don't understand why `any` is needed here.